packages/components/tree-v2/src/composables/useTree.ts

```diff
@@ -191,7 +191,7 @@ export function useTree(props: TreeProps, emit) {
     }
   }
 
-  function setExpendKeys(keys: TreeKey[]) {
+  function setExpandKeys(keys: TreeKey[]) {
```

Comment:
`setExpandedKeys` will be better. `Expand` is a verb.

Reply:
Okay!